app/models/lockstep/connection.rb

```diff
@@ -11,10 +11,11 @@ class Lockstep::Connection < Lockstep::ApiRecord
 
   validates :company_name, presence: true
 
-  default_scope { where(company_type: %w[Customer Vendor Group Company]).or(where(company_type: nil)) }
+  default_scope { where(company_type: %w[Customer Vendor Group Company CompanyProfile]).or(where(company_type: nil)) }
 
   scope :customers, -> { where(company_type: 'Customer') }
   scope :vendors, -> { where(company_type: 'Vendor') }
   scope :internal_connections, -> { where(company_type: 'Company') }
+  scope :profile_companies, -> { where(company_type: 'CompanyProfile') }
```

Comment:
nitpick: IMO, a better scope name would be `company_profiles`

Reply:
Scope renamed